### PR TITLE
chore: add avm files

### PR DESCRIPTION
**Pull request by luke-taylor**
## Issue number

#000

## Checklist before requesting a review
- [ ] The pr title can be used to describe what this pr did in `CHANGELOG.md` file
- [ ] I have executed pre-commit on my machine
- [ ] I have passed pr-check on my machine

Thanks for your cooperation!

### CODE REVIEW

**Comment by luke-taylor** [comment]
@matt-FFFFFF @lonegunmanb 

Moving an old TFVM module to AVM, I used the grept apply to get the main files, then added the correct 1ES pool etc.

**Review by matt-FFFFFF — APPROVED** [review]

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Support
+
+> ⚠️**Note:** For the full details on the support statements, SLAs, and more for the Azure Verified Modules (AVM) initiative please visit [aka.ms/AVM/Support](https://aka.ms/avm/support) ⚠️
+
+## How to file issues and get help
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the existing issues before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new issue.
+
+Issues can be created and searched through for existing [issues here](https://github.com//issues).
```
Comment (matt-FFFFFF) [inline SUPPORT.md:9]: Change this. It needs the GITHUB_REPOSITORY Env var set

**Review by matt-FFFFFF — CHANGES_REQUESTED** [review]

**Comment by matt-FFFFFF** [comment]
Sorry one small change

**Review by matt-FFFFFF — APPROVED** [review]
LGTM

**Pull request merged**